upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,11 @@
 == Migration Changes
 
+=== Migrating to 12.0.0
+
+==== Skip creation of user session for the Docker protocol authentication
+
+There is no user session created after successful authentication with the Docker protocol. For details, please refer to the link:{adminguide_link}#instagram[{adminguide_name}].
```

Comment:
I guess the link should point to `#_docker`?

Reply:
Oops, Thanks for catch this!